CMakeLists.txt

```diff
@@ -18,6 +13,15 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_GLIBCXX_USE_CXX11_ABI=0")
 # We require a C++11 compliant compiler
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# ray build options
+option(RAY_BUILD_PYTHON
+    "build python library"
+    OFF)
```

Comment:
Should be "ON" by default?

Reply:
OK. Actually it was `ON`...

Reply:
I just found here it needs to keep OFF. So to keep consistent with build.sh.

If we set it `ON`, `./build.sh -l java` will always build python language. So I keep it `OFF`